### ISSUE TITLE
There is a website called Jango that NVDA does not work with very well.  NVDA doesn't detect/read some of the buttons.  Jango is a music website similar to Pandora, and it is accessible with JAWS, but not NVDA.  It is a little more accessible when I use NVDA with Internet Explorer 9, but not Mozilla Firefox.  http://www.jango.com

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2654]
**Reported by victor68 on 2012-09-07 19:02**
NVDA cannot read the "Thumbs Up" or the "Thumbs Down" buttons on the Jango website.  There are other buttons it cannot read either.  http://www.jango.com

**Comment by leonardder** [comment]
This website is likely to have changed, also browsers and NVDA have changed/improved things. Closing, as this website overall looks not too bad. See also #1904. Please comment if you feel there is still an issue with this website. If so, please elaborate on why and how you think the problems are NVDA specific.
